blueflood-http/src/test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpEventsIngestionHandlerTest.java

```diff
@@ -87,34 +86,24 @@ public void testElasticSearchInsertCalledWhenPut() throws Exception {
         verify(searchIO).insert(TENANT, events);
     }
 
-    @Test public void testMalformedEventPut() throws Exception {
+    @Test
+    public void testMalformedEventPut() throws Exception {
         final String malformedJSON = "{\"when\":, what]}";
         handler.handle(context, createRequest(HttpMethod.POST, "", malformedJSON));
         ArgumentCaptor<DefaultHttpResponse> argument = ArgumentCaptor.forClass(DefaultHttpResponse.class);
         verify(searchIO, never()).insert(anyString(), anyList());
         verify(channel).write(argument.capture());
-        Assert.assertNotSame(argument.getValue().getContent().toString(Charset.defaultCharset()), "");
+        assertNotSame(argument.getValue().getContent().toString(Charset.defaultCharset()), "");
     }
 
-    @Test public void testMinimumEventPut() throws Exception {
+    @Test
+    public void testMinimumEventPut() throws Exception {
         Map<String, Object> event = new HashMap<String, Object>();
         event.put(Event.FieldLabels.data.name(), "data");
         ArgumentCaptor<DefaultHttpResponse> argument = ArgumentCaptor.forClass(DefaultHttpResponse.class);
         handler.handle(context, createPutOneEventRequest(event));
         verify(searchIO, never()).insert(anyString(), anyList());
         verify(channel).write(argument.capture());
-        Assert.assertEquals(argument.getValue().getContent().toString(Charset.defaultCharset()), "Invalid Data: Event should contain at least 'what' field.");
-    }
-
-    @Test public void testApplyingCurrentTimeWhenEmpty() throws Exception {
-        Map<String, Object> event = createRandomEvent();
```

Comment:
We are not allowing empty timestamps.  Previously, if no timestamp, it was treated as a timestamp of 0.